pynsee/localdata/get_local_data.py

```diff
@@ -146,8 +148,9 @@ def get_local_data(
                 update=True,
             )
         else:
-            print(
-                f"Locally saved data has been used\nSet update=True to trigger an update"
+            logger.info(

```

Comment:
cc @hadrilec (info vs warning)

Reply:
cf. supra

Reply:
no need to reply every time ;) I needed to comment everywhere at the beginning so people could easily locate the conversation